src/Fetcher/JoindApiFetcher.php

```diff
@@ -203,6 +209,7 @@ public function denormalizeConference(array $rawConference): ?Conference
         return $conference;
     }
 
+    /** @return array<array>\null */
```

Comment:
...

Reply:
I left these on purpose because we were returning null explicitely in the method, but we could return an empty array (which I considered but didn't really like)